<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OfferWidget</class>
 <widget class="QWidget" name="OfferWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="leFilter">
     <property name="placeholderText">
      <string>Filter...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_tbl">
     <item>
      <widget class="QTableView" name="tblOffers">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icon/edit</normaloff>:/icon/edit</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrint">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Print</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icon/print</normaloff>:/icon/print</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Export to PDF</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icon/save</normaloff>:/icon/save</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icon/delete</normaloff>:/icon/delete</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
